lib/compat/wordpress-5.9/class-wp-theme-json-5-9.php

```diff
@@ -555,7 +555,7 @@ protected static function get_blocks_metadata() {
 			foreach ( static::ELEMENTS as $el_name => $el_selector ) {
 				$element_selector = array();
 				foreach ( $block_selectors as $selector ) {
-					$element_selector[] = $selector . ' ' . $el_selector;
+					$element_selector[] = ':where(' . $selector . ' ' . $el_selector . ')';
```

Comment:
Do you think this will make it into WordPress 6.0?

Unless there's something I'm missing, I think the general approach for migration is to move the method (`get_blocks_metadata()`) into[ class-wp-theme-json-gutenberg.php](https://github.com/WordPress/gutenberg/blob/eb4fc8afa3576e3956938586e642b68430de7ed1/lib/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php) and modify it there.

Reply:
>Also, this is only in the 5.9 compat code for ease of quick setup and test - if this approach moves forward it would need to be migrated to 6.0+ compat code

Noted that in a comment above  - just taking the quick and dirty approach for starters and will migrate if there is support for this approach.

Reply:
Oh, apologies. I started looking at the code first. 😇 Thanks for the comment.